src/js/components/Calendar/stories/Dual.js

```diff
@@ -42,9 +34,11 @@ export const Dual = () => {
         onReference={(reference) => {
           const refDate = new Date(reference);
           const nextDate = new Date(refDate);
+          console.log(refDate);
+          console.log(nextDate);
           nextDate.setMonth(refDate.getMonth() + 1, 1);
-          setReference1(refDate.toISOString());
-          setReference2(nextDate.toISOString());
+          setReference1(refDate.toString());
```

Comment:
This seems incorrect to me. I think `toISOString` would be better here.

Reply:
fixed in latest commit